src/models.py

```diff
@@ -3,7 +3,7 @@
 
 from pytorch_lightning import LightningModule
 from torch import Tensor
-from torchscale.architecture.config import RetNetConfig, DecoderConfig
+from torchscale.architecture.config import RetNetConfig, DecoderConfig, LongNetConfig
```

Comment:
If you have a second, it would be nice to have this list of imports alphabetized like the comment above.

Reply:
Fixed in 6ce6d52.